src/components/Button/ButtonHW.css

```diff
@@ -0,0 +1,82 @@
+.BTNOne {
```

Comment:
Название основного класса должно совпадать с названием компонента в нижнем регистре. 
```suggestion
.button {
```